opba-auth-rest-impl/src/main/java/de/adorsys/opba/tppauthapi/filter/CookieRenewalFilterConfig.java

```diff
@@ -0,0 +1,23 @@
+package de.adorsys.opba.tppauthapi.filter;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Slf4j
+@Configuration
+public class CookieRenewalFilterConfig {
```

Comment:
We already have Filters defined in the security module. It makes sense to place both CookieRenewalFilterConfig and CookieRenewalFilter there. Also, it makes sense to protect entire consent API with this filter too

Reply:
Moved filters. And Key from HttpHeaders to security module. But did not protect entire consent API